README.md

```diff
@@ -143,6 +143,24 @@ brew install nginx-full --with-tarantool-module
 
 [Back to contents](#contents)
 
+## Run tests

```

Comment:
Please, add into the table of contents.

I would also add the 'Back to contents' link as in all other sections.

Reply:
Done.